src/components/Details/MinutesItem/MinutesItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+const DocumentsListOpen = styled.summary({
+  width: "7.5rem",
+});
+
+const DocumentsListParagraph = styled.p({
+  textDecoration: "underline",
+});
+
+interface Docs {
+  /*Document item label */
+  label: string;
+  /*Document item url */
+  url: string;
+}
+
+interface Items {
+  /*Minute item headline */
+  item: string;
+  /*Array of documents to be shown below*/
+  docs?: Docs[];
+}
+
+export interface MinutesItemProps {
+  minutesItems: Items[];
+}
+
+function returnDocList(docs: Docs[] | undefined) {
+  if (docs) {
+    return (
+      <section>
+        <details>
+          <DocumentsListOpen>
+            <DocumentsListParagraph>See documents</DocumentsListParagraph>
+          </DocumentsListOpen>
+          {docs.map((doc) => {
+            return (
+              <div key={doc.label}>
+                <a href={doc.url}>{doc.label}</a>
+              </div>
+            );
+          })}
+        </details>
+      </section>
+    );
+  }
+}
+
+const MinutesItem: FC<MinutesItemProps> = ({ minutesItems }: MinutesItemProps) => {
+  return (
+    <ol className="mzp-u-list-styled">
+      {minutesItems.map((elem) => {
+        return (
+          <li key={elem.item}>
+            <div>{elem.item}</div>
+            <div>{returnDocList(elem.docs)}</div>
+          </li>
+        );
+      })}
+    </ol>
+  );
+};
+
+export default MinutesItem;
```

Comment:
My nitpick is to ask to rename the prop from `minutesItems` to `attachments`? Because the number list of this component is the full "Minutes Item List" where each minutes item _may have_ some attachments.

Reply:
I think you mean change `Docs` to `attachments`, right?

Reply:
uhhhhhhhh woops. Alright it's not the prop that is confusing me its the name of the component. Can we name it "MinutesItemsList"? A single MinuteItem isn't what we are doing here, this is the full list right?

Reply:
Yes, that's right, this is the full list. So yes, good call on MinutesItemsList